clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -201,6 +201,13 @@ public String getString(String key) {
         return configKey.type;
     }
 
+    public String documentationOf(String key) {
```

Comment:
Since this is a public API, we should include this change in the KIP.

Reply:
Done https://cwiki.apache.org/confluence/display/KAFKA/KIP-569%3A+DescribeConfigsResponse+-+Update+the+schema+to+include+additional+metadata+information+of+the+field#KIP-569:DescribeConfigsResponse-Updatetheschematoincludeadditionalmetadatainformationofthefield-AbstractConfigClass

Reply:
We should also send a note to the VOTE thread on the mailing list to mention that the KIP was updated.